DirectEmployers/MyJobs
# [PD-2297] Export Screen

## Test:

1. Run a report.
2. Export it. Behold the new export options screen.
3. Play with the options to look for problems. Everything should work. It should look and behave as designed by @misterjonscott. (Except for the fact that the column names are ugly)

**Note:** I don't recall every talking about where in the data models we put user visible names for columns *as they appear in the report*. This is mentioned in PD-2293.

## Review:

Some previous discussion happened in another PR: https://github.com/DirectEmployers/MyJobs/pull/2245

As far as I can tell in that discussion there wasn't actionable feedback except for a broken dependency. Please point out if I missed something from that PR.
